route.js

```diff
@@ -0,0 +1,65 @@
+const express = require("express");
```

Comment:
Prefer not to mix `require` with `import`. Let's stick with one: `import`
```suggestion
import express from "express";
```